sdcm/nemesis.py

```diff
@@ -82,9 +82,8 @@ def update_stats(self, disrupt, status=True, data={}):
         if self.tester.create_stats:
             self.tester.update({'nemesis': self.stats})
 
-    def publish_event(self, disrupt, status=True, data={}):
-        data['node'] = self.target_node
-        DisruptionEvent(name=disrupt, status=status, **data)
+    def publish_event(self, type, disrupt, status=True, data={}):
```

Comment:
why do we need `publish_event` on Nemesis class, we can just use DisruptionEvent directly
(it also annoying when we have 2 `publish_event` methods and finding method definition in PyCharm)

Reply:
done